utils/cookies.js

```diff
@@ -0,0 +1,30 @@
+export function setCookie(name, value, options = {}) {
```

Comment:
Do you think the cookie needs to be encrypted?

Reply:
Yes, you're right, the cookie should not be available to the client JS. I changed the approach to set an httpOnly cookie server-side, which is the same approach as `nextjs-auth0`, so I think that should be fine then 👍